api.yaml

```diff
@@ -327,41 +236,203 @@ paths:
             type: string
         - name: start
           in: query
-          description: Start of the date range in ISO format YYYY-MM-DD
+          description: Start of the date range in ISO format `YYYY-MM-DD`
           required: true
           schema:
             type: string
             format: date
             example: "2019-06-21"
         - name: end
           in: query
-          description: End of the date range in ISO format YYYY-MM-DD
+          description: End of the date range in ISO format `YYYY-MM-DD`
           required: true
           schema:
             type: string
             format: date
             example: "2019-06-21"
       responses:
         "200":
-          description: List of timeslots per day
+          description: Available variants by date and timeslots
           content:
             application/json:
               schema:
-                $ref: "#/components/schemas/Timeslots"
+                $ref: "#/components/schemas/AvailabilityResponse"
+              examples:
+                timeslotted:
+                  summary: Available variants for different timeslots per day
+                  value:
+                    "2022-12-19T11:00":
+                      available_tickets: 0
+                      variants:
```

Comment:
Check if we can specify an empty list in YAML without having to list all the variants.

Reply:
If there is no availability it would be better to show that they can returns such timeslot without variants than skipping it entirely.